package.json

```diff
@@ -19,6 +19,10 @@
     "elysia": "^0.7.12",
     "typescript": "^5.2.2"
   },
+  "dependencies": {
+    "@elysiajs/cookie": "^0.7.0",
+    "@elysiajs/jwt": "^0.7.0"
```

Comment:
This is a bad practice. There is an update work in progress that will fix this "problem" described in https://github.com/bogeychan/elysia-oauth2/issues/1.

Reply:
Fixed.

Reply:
@bogeychan in this update work are u using cookies to store data?

Reply:
I've made a change into a fork of mine [elysia-oauth2](https://www.npmjs.com/package/elysia-oauth2) just to use in a project I'm working at and changed to make the full req available through storage.


This way, with the usage of cookies and jwt an example code will look like this:

```ts
storage: {
    // storage of users' access tokens is up to you and you can access it using cookies
    async get(req, name) {
      return req.jwt.verify(req.cookie[`${name}.session`]);
    },
    async set(req, name, token) {
      req.setCookie(`${name}.session`, await req.jwt.sign(token));
      return token;
    },
    async delete(req, name) {
      req.removeCookie(`${name}.session`)
    }
  }
```

The bad practice came in reuse an standard lib? (I'm asking for learning purposes!)

Reply:
I got it! That's a good take! 😄